Bizware/BizHawk.Bizware.BizwareGL/TexAtlas.cs

```diff
@@ -59,106 +59,105 @@ public static PackedAtlasResults PackAtlas(IEnumerable<RectItem> items)
 			List<RectItem> currentItems = new List<RectItem>(items);
 			List<RectItem> remainItems = new List<RectItem>();
 
-		RETRY:
-
-			//this is where the texture size range is determined.
-			//we run this every time we make an atlas, in case we want to variably control the maximum texture output size.
-			//ALSO - we accumulate data in there, so we need to refresh it each time. ... lame.
-			List<TryFitParam> todoSizes = new List<TryFitParam>();
-			for (int i = 3; i <= MaxSizeBits; i++)
-			{
-				for (int j = 3; j <= MaxSizeBits; j++)
-				{
-					int w = 1 << i;
-					int h = 1 << j;
-					TryFitParam tfp = new TryFitParam(w, h);
-					todoSizes.Add(tfp);
-				}
-			}
-
-			//run the packing algorithm on each potential size
-			Parallel.ForEach(todoSizes, (param) =>
+			while (true)

```

Comment:
Can't comment how well this one handles the job (@zeromus required), but looks reasonable at least.